packages/validation/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default } from './config';
```

Comment:
please no default again :D this is really ugly, named export please

Reply:
I want yup to be a default export since it's the centerpiece of the package, like React.

Reply:
There is no reason to use default export even when it is centerpiece. It hard to even define what is centerpiece. Here is quite obvious but you could have other packages where it won't be or where you don't want default at all (components libs for example). That leads to confusing DX where you have to guess which package have default export, which not and what could be exported default.

Default exports are bad for other many reasons, it only supports chaos in exports/imports. If everything is named export then you don't have to think about it. You can check lot of famous libraries and they are using just two patterns of exports:

1. Default exports that serves as wrapper everything - React, lodash etc.
2. Just named exports - redux, ramda, apollo-client, graphql, react-hook-form, redux-forms... nearly all solid libraries

First pattern is not completely bad for some usages, but default exports leads to implicit imports names, worse DX, worse autocomplete... https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad

Default exports is another JS magic that was IHMO mistake and never should be introduced. There is nothing like default exports in other solid languages and I believe libraries like `React` or `yup` supports default exports only because historical compatibility reason and potential huge breaking change. So we should not repeat their mistakes.

Reply:
done